atlassian/bitbucket/cloud/base.py

```diff
@@ -85,3 +97,10 @@ def get_data(self, id, default=None):
 
     def get_link(self, link):
         return self.__data["links"][link]["href"]
+
+    def get_time(self, id):
+        value_str = self.get_data(id)
+        if not value_str or not isinstance(value_str, str):
```

Comment:
IF the value is `False` the test is true and None is returned.
```suggestion
        if value_str is None or not isinstance(value_str, str):
```

Reply:
But if it's `False` than it's `bool` and the second condition will trigger...this implementation will even solve the falsy string problem when the `string == ""`

Reply:
When is a time string empty? I think in this case the exception from parsing the time is more suitable. Also if there is a empty string and the user configured raw values, this empty string should be returned.